src/index.html

```diff
@@ -2,9 +2,10 @@
 <html lang="en">
 <head>
   <meta charset="utf-8">
-  <title>MSSE663S8W1GroupProject</title>
+  <title>MSSE663 Recipe App</title>
```

Comment:
Class name may not be necessary

Reply:
I totally agree! Communication has been low and I wanted to plant the idea of changing these little things to make it our own instead of just going ahead and doing it without asking others. Thanks for noticing!